app/app.go

```diff
@@ -467,6 +469,8 @@ func (app *SifchainApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain)
 		panic(err)
 	}
 
+	genesisState["ethbridge"] = json.RawMessage{}
```

Comment:
This is strange.  It's just a hack to get InitGenesis called - if the genesis data for "ethbridge" is empty, InitGenesis isn't called at all, and that's where we set up:

```
	bridgeAccount := authtypes.NewEmptyModuleAccount(ModuleName, authtypes.Burner, authtypes.Minter)
	am.AccountKeeper.SetModuleAccount(ctx, bridgeAccount)
```

Reply:
don't understand, why just clean the the genesis state of this module, if also need clean other module like oracle, clp.

Reply:
This was removed.